src/components/not-found.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { NimbleEmoji } from '.'
+
+export default class NotFound extends React.PureComponent {
+  constructor(props) {
+    super(props)
+    this.state = { errored: false }
+
+    this.handleError = this.handleError.bind(this)
+  }
+
+  defaultImage() {
+    var { data, emojiProps } = this.props
+
+    return NimbleEmoji({
+      data: data,
+      ...emojiProps,
+      size: 38,
+      emoji: 'sleuth_or_spy',
+      onOver: null,
+      onLeave: null,
+      onClick: null,
+    })
+  }
+
+  handleError(event) {
+    this.setState({ errored: true })
+  }
+
+  render() {
+    var { i18n, notFound } = this.props
+
+    if (this.state.errored) {
```

Comment:
Rather than doing going down the route of letting `img` error in the default case (no valid `notFoundImg` set) and then falling back to this, should we just check if `notFoundImg` is empty or null and run the default? 

That'd save us a render and a `setState`